### PR TITLE
Make Simple Improvements to Engineering Diffraction 2 GUI

### LINKED ISSUE #27724: Make Improvements from Engineering Meeting
### Expected behavior
The "Peaks Fitted" line should only show markers, not lines. 
The output file for GSAS `.prm` files should be at most 80 characters in each line. 
Tabbing order is correct
### Actual behavior
Both lines have the line and markers drawn. 
Output files contain file paths that are too long.
Tabbing order is incorrect
### Steps to reproduce the behavior
Create a new calibration in the Engineering Diffraction 2 GUI and plot the output. 
Check the produced file. 
### Platforms affected
All

### PR DESCRIPTION
**Pull request by cailafinn**
### ~Not for review until #27749  is merged~

**Description of work.**
Fixed some issues in response to comments made by instrument scientists in last meeting
- Fixed tabbing order of Calibration and Focus tabs. 
- Removed lines from raw data calibration output plot lines. 
- Reverted GSAS `.prm` output files to using run numbers instead of file paths to keep line length under 20 characters. 

**To test:**
1. Open Engineering Diffraction 2 GUI
2. Test the tabbing order works as you would expect. 
2. Create a calibration and plot outputs (V: 307521, Sample (CeO2): 305738)
3. Check that the peaks fitted data is only made up of makers, without lines.  
4. Check that none of the lines in the generated `.prm` file exceed 80 characters. (if you didn't have a save location saved, it should have appeared in `~/Engineering_Mantid/Calibration`). 
This is a requirement from GSAS, one of the other programs used by the instrument scientists. 

Fixes #27724 

*This does not require release notes* because **(as always) the interface is not user facing yet.**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Review by Harrietbrown — DISMISSED** [review]
The Tabs are in the correct order, the output files have all lines less than 80 characters long, and the plots are all as described. Good Job.

File: qt/widgets/common/inc/MantidQtWidgets/Common/MWRunFiles.ui [hunk 1]
```diff
@@ -149,13 +149,16 @@
       <string/>
      </property>
      <property name="icon">
-      <iconset resource="../../../../images/images.qrc">
```
Comment (martyngigg) [inline qt/widgets/common/inc/MantidQtWidgets/Common/MWRunFiles.ui]: Can I ask the reason that the resource file is removed? Do we not need it for the `LoadFile.png` reference?

Comment (ConorMFinn) [reply qt/widgets/common/inc/MantidQtWidgets/Common/MWRunFiles.ui]: This appears to be a mistake from the changes made in QtDesigner. I'll revert this like back.

Comment (martyngigg) [reply qt/widgets/common/inc/MantidQtWidgets/Common/MWRunFiles.ui]: Thanks!

**Review by martyngigg — APPROVED** [review]

**Pull request merged**